pkg/database/sms_signing_key.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package database
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/jinzhu/gorm"
+)
+
+var _ ManagedKey = (*SMSSigningKey)(nil)
+
+// SMSSigningKey represents a reference to a KMS backed signing key
+// version for SMS payload signing.
+type SMSSigningKey struct {
+	gorm.Model
+	Errorable
+
+	// A signing key belongs to exactly one realm.
+	RealmID uint `gorm:"index:realm"`
+
+	// Reference to an exact version of a key in the KMS
+	KeyID  string
+	Active bool
+}
+
+// GetKID returns the 'kid' field value to use in signing JWTs.
+func (s *SMSSigningKey) GetKID() string {
+	return fmt.Sprintf("r%dv%d", s.RealmID, s.ID)
```

Comment:
This doesn't seem to match `realm.SMSSigningKeyID()`?

Reply:
SMSSigningKeyID is the KMS key - this is the KID for public consumption